### ISSUE TITLE
Embeddings not actually in repo?

### ISSUE THREAD

**Issue by robbymeals** [junhua/IPOD#2]
Hey, just curious, I know there's the checkpoint file for download and so anyone should be able to recreate them, but the embeddings aren't actually included in the repo here. Is that intentional? 
Thanks!

**Comment by choprashweta** [comment]
Did you manage to figure out how to recreate the embeddings? Looking to do the same.

**Comment by SM-JM** [comment]
I agree. Also I have been trying for days to try and import the model into allannlp but I can't figure out which model to do so, all the emlo related models require weight files as well
